OpenEdXMobile/src/test/java/org/edx/mobile/test/ConfigTests.java

```diff
@@ -5,16 +5,18 @@
 import com.google.gson.JsonPrimitive;
 
 import org.edx.mobile.util.Config;
-
+import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 import org.junit.runners.Parameterized.Parameters;
-import org.junit.Test;
 
-import java.util.Arrays;
 import java.util.ArrayList;
+import java.util.Arrays;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
```

Comment:
Auto-import doesn't seem to do this for me. What's the command you use which does this?

Anyway, it's up to you; personally I don't like automation tools make usability-level decisions 😜

Note that you can set a threshold in Android Studio on the number of (static and non-static) imports that are condensed to a wildcard, but this actually needs to be decided on a case-by-case-basis.